src/Components/benchmarkapps/Wasm.Performance/TestApp/wwwroot/benchmarks/jsonHandlingStress.js

```diff
@@ -0,0 +1,44 @@
+import { group, benchmark, setup, teardown } from './lib/minibench/minibench.js';
+import { benchmarkJson} from './jsonHandlingData.js';
+import { BlazorStressApp } from './util/BlazorStressApp.js';
+
+group('JSON handling', () => {
+  let app;
+
+  setup(async () => {
+    app = new BlazorStressApp().app;
+    app.navigateTo('json');
+  });
+
+  benchmark('Serialize 1kb', () =>
```

Comment:
We can be more selective about what gets run as part of stress. For instance, we no longer track download sizes or JSON serialization using JavaScript since those aren't useful stress scenarios. One fallout of this change is that there isn't a good way to "stress test" time to first UI any longer. We could create a separate instance of the blazor app, but it's not clear how that helps identifying stress issues.